src/stores/Video.js

```diff
@@ -360,6 +350,42 @@ class VideoStore {
     }
   });
 
+  SetSupportedOffering = flow(function * ({versionHash, browserSupportedDrms}) {
```

Comment:
this should be called GetSupportedOffering because it doesn't actually set anything